### ISSUE TITLE
Windows 10 not displaying theme

### ISSUE THREAD

**Issue by Niightmare** [LazoCoder/Pokemon-Terminal#165]
Help!

I followed the windows tutorial but when I type "pokemon pikachu" on cmder, it doesn't display anything

![image](https://user-images.githubusercontent.com/20825412/53936241-da9fd980-406e-11e9-96e5-e35ad3c1b3d8.png)

**Comment by jimmyorourke** [comment]
did you follow these steps https://github.com/LazoCoder/Pokemon-Terminal#conemu-settings

**Comment by Niightmare** [comment]
Yes, I followed every single steps but still not working even I reinstalled everything

**Comment by jimmyorourke** [comment]
if you go to the settings menu, which ConEmu version does it say in the top bar?

**Comment by ayoub-bousetta** [comment]
Mine said This terminal emulator is not supported.

**Comment by jimmyorourke** [comment]
install using pip, not npm. I'm not sure who manages the npm package but it doesn't pull right from github and i'm not sure it's been updated

**Comment by sylveon** [comment]
@LazoCoder manages npm

**Comment by LazoCoder** [comment]
I've updated the npm package.

It should be good now. Let me know if there is still a problem.

**Comment by Niightmare** [comment]
Hi @LazoCoder 
I'm still trying with npm but it's displaying me this 
![image](https://user-images.githubusercontent.com/20825412/55527643-2e461880-5657-11e9-9581-4bf2ef39feba.png)

cmder v. 180626
python v. 3.7.2

**Comment by LazoCoder** [comment]
It's looking for a file that used to be in this repo but was removed.

Create a file "pokemon" in the path that is in your console: C:\Users\...\node_modules\pokemon-terminal

And copy and paste this into that file: https://github.com/LazoCoder/Pokemon-Terminal/blob/3f29dd84239a8ac4110270408f27e8c0945953fb/pokemon

You may want to change "#!/usr/bin/env python3.6" to "#!/usr/bin/env python3.7".

Let me know if this works.

**Comment by LazoCoder** [comment]
Alternatively, you can go into package.json and change:
```
  "bin": {
    "pokemon": "pokemon",
    "ichooseyou": "pokemon"
  },
```
to
```
  "bin": {
    "pokemon": "main.py",
    "ichooseyou": "main.py"
  },
```
Try that ^ solution first. If it works, let me know so I can update package.json in the repo.

**Comment by Niightmare** [comment]
Sorry for the late, but i tried downloading again and creating the missing folder with no success. When its downloading it removes the folder created and causes this problem

> Hi @LazoCoder
> I'm still trying with npm but it's displaying me this
> ![image](https://user-images.githubusercontent.com/20825412/55527643-2e461880-5657-11e9-9581-4bf2ef39feba.png)
> 
> cmder v. 180626
> python v. 3.7.2

**Comment by SchemeSonic** [comment]
Same problem with @Niightmare 

```
➜  Desktop python --version
Python 2.7.15
➜  Desktop python3 --version
Python 3.7.3
➜  Desktop npm --version
6.9.0
➜  Desktop node --version
v8.11.3
```

**Comment by luturol** [comment]
> You may want to change "#!/usr/bin/env python3.6" to "#!/usr/bin/env python3.7".

I've solved the first issue downloading the source and executing `npm install --global` but now when i call `pokemon pikachu` return the message bellow
![image](https://user-images.githubusercontent.com/9202337/59397554-62c70680-8d63-11e9-8f7e-067a51ea2ca1.png)

**Comment by jimmyorourke** [comment]
Please try installing with `pip` (or `pip3`) folks. `pip` should be your go-to for installing python based tools/packages.

**Comment by sylveon** [comment]
Also, I noticed that while the background is set, it is not enabled by default, which causes the original issue that @Niightmare had

![image](https://user-images.githubusercontent.com/6440374/59442350-106c0100-8dc8-11e9-8a85-9f9e43fa9656.png)

**Comment by jimmyorourke** [comment]
@sylveon are you saying there is something that is not being enabled properly in the code, or that the readme instructions need an update, or something else?
Feel free to open a PR if updates are needed somewhere (with screenshots if helpful for the instructions)

**Comment by sylveon** [comment]
I'm currently looking if it's possible to somehow programmatically toggle this checkbox.

**Comment by luturol** [comment]
Hi, I`m having this problem trying to use pokemon-terminal. Already tried to use Pip, but it still keep me returning this error

![image](https://user-images.githubusercontent.com/9202337/59476822-e991e700-8e28-11e9-8920-1cbcdb3f0307.png)

Can you guys help me?

**Comment by luturol** [comment]
Hi, I've already installed Pip and installed pokemon-terminal from pip, but its keep returning this error
![image](https://user-images.githubusercontent.com/9202337/59476930-51483200-8e29-11e9-93bc-1b333fc09091.png)

Can you guys help me?

**Comment by jimmyorourke** [comment]
@luturol are you sure the npm installed version was uninstalled properly?
